src/left-nav-title.json

```
@@ -715,7 +715,7 @@
   "testrail": {"/docs/integrations/test-management/testrail/": "TestRail"},
   "label-management": {"/docs/test-cases/manage/label-management/": "Label Management" },
   "mobile-accessibility-testing": {"/docs/accessibility-testing/mobile-accessibility-testing/": "Accessibility Testing for Android & iOS"},
-  "live-editor": {"/docs/live-editor/": "Testsigma Terminal (Live Editor)" },
+  "live-editor": {"/docs/copilot/": "Testsigma Terminal (Live Editor)" },
```

Biome (2.4.16)

[error] The key live-editor was already declared.

(lint/suspicious/noDuplicateObjectKeys)